File: naughty_nice.py
def what_list_am_i_on(actions):
    naughty_count = 0
    nice_count = 0
    for x in actions:
        if x[0] == 'b' or x[0] == 'f' or x[0] == 'k':
            naughty_count += 1
        elif x[0] == 'g' or x[0] == 's' or x[0] == 'n':
            nice_count += 1
    if naughty_count >= nice_count:
        return 'naughty'
    else:
        return 'nice'

File: naughtyNice.js
function whatListAmIOn(actions){
    let naughtyCount = 0;
    let niceCount = 0;
   for (let i = 0; i < actions.length; i++){
     if (actions[i][0] == 'b' || actions[i][0] == 'f'||actions[i][0] == 'k'){
       naughtyCount++;
     }else if (actions[i][0] == 'g'||actions[i][0] == 's'||actions[i][0] == 'n'){
       niceCount++;
     }
   }
    if (naughtyCount >= niceCount){
      return 'naughty';
    }
    return 'nice';
  }